test/sequential/test-child-process-execsync.js

```diff
@@ -37,12 +37,10 @@ let caught = false;
 assert.throws(
   function() { execSync('exit -1', { shell: 'bad_shell' }); },
   /spawnSync bad_shell ENOENT/,
```

Comment:
Can you remove the trailing comma here and below.

Reply:
Removed trailing commas.